src/actions/comments.js

```diff
@@ -43,12 +44,23 @@ export function appendComment (postId, comment) {
   }
 }
 
-export function createComment ({ postId, text, tagDescriptions, imageUrl }) {
+export function createComment ({ postId, text, tagDescriptions, imageUrl, userId }) {
   const params = {text, tagDescriptions, imageUrl}
+  const comment = {
```

Comment:
let's call this `optimisticComment` or `tempComment`? it's a little confusing otherwise to see in the reducer that the comment creation has successfully completed, and so we should _remove_ something matching `comment.id`